meta/mm_sequence.py

```diff
@@ -117,8 +119,23 @@ def update_rqmt(rqmt, extra):
 
         for a_idx, action in enumerate(action_sequence):
             if action.startswith("align"):
+                if use_corrected_applicator:
+                    from i6_core.am.config import adjust_am_config_for_corrected_applicator
+                    from i6_core.rasr.crp import CommonRasrParameters
+                    import copy
+
+                    align_crp = CommonRasrParameters(base=crp)
+                    adjust_am_config_for_corrected_applicator(align_crp.acoustic_model_config)
+                    align_crp.acoustic_model_config = copy.deepcopy(crp.acoustic_model_config)
+                    align_crp.acoustic_model_config["*"].allow_for_silence_repetitions = False
+                    align_crp.acoustic_model_config["*"].normalize_lemma_sequence_scores = False
+                    align_crp.acoustic_model_config["*"].fix_allophone_context_at_word_boundaries = True
+                    align_crp.acoustic_model_config["*"].transducer_builder_filter_out_invalid_allophones = True
```

Comment:
can we find and use the full identifier?

Reply:
Currently, this is already under a very long identifier, not sure if it is a good idea to make it even longer. I can check if you think this is necessary

Reply:
I see, thanks for the suggestion and the hint. I will check this out and submit the correction,